src/main/java/wolox/training/constants/ExceptionConstants.java

```diff
@@ -0,0 +1,6 @@
+package wolox.training.constants;
+
+public class ExceptionConstants {
```

Comment:
Excellent idea! A good thing to add here is:
1) Define this class as final (Which means forbid inheritance) 
2) Provide a private constructor to overrides the default one, this way we avoid instantiation (We only want this class to be used as a static class):
```
private void ExceptionConstants() {

}
```